config/locales/en.yml

```
@@ -675,6 +685,32 @@ en:
   user_missing_organisation:
     body: "%{contact_link} to assign an organisation to your account."
     title: You do not have an organisation set for your account
+  user_upgrade_request:
+    new:
+      body_html: |
+        <p>GOV.UK Forms is in development with selected partners in central government.</p>
+
+        <p>If you’re from a central government organisation you might be able to upgrade to an editor account early. You’ll need to meet some requirements.</p>
+
+        <p>You may be able to upgrade to an editor account if:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>you need to make an online form to be published on GOV.UK</li>
+          <li>you believe your form can be created with the features already available — <a href="https://www.forms.service.gov.uk/features" class="govuk-link">read about the features available already</a> and try to make the form to check this</li>
+          <li>your form is expected to get between 250 and 10,000 submissions a year</li>
+          <li>you're happy to be asked to provide feedback and take part in user research</li>
```

Tiny thing but we can use curly quotes here:
```suggestion
          <li>you’re happy to be asked to provide feedback and take part in user research</li>
```